parts/plugins/x_builddeb.py

```diff
@@ -54,7 +54,7 @@ def build(self):
             # because quite a few of them will break
             env["DEB_BUILD_OPTIONS"] = "nocheck"
         # run the real build
-        self.run(["dpkg-buildpackage"], env=env)
+        self.run(["dpkg-buildpackage", "-b", "-Zgzip"], env=env)
```

Comment:
this is an opportunistic change that makes the `snapd-snap` test go from 300s to 90s

Reply:
Awesome speedup. Would it make sense to push it as a separate PR?

Reply:
It would, but it's also small enough (and side-effect-less enough) that I don't think it hurts here

Reply:
OTOH there's another dpkg-buildpackage that's doing xz compression somewhere, that would be nice to find and kill; this and that could be part of a single pr